medusa/providers/nzb/anizb.py

```diff
@@ -118,5 +121,30 @@ def _get_size(self, item):
         """Override the default _get_size to prevent it from extracting using the default tags."""
         return try_int(item.get('size'))
 
+    def _get_episode_search_strings(self, episode):
```

Comment:
I would ignore it in codacy. There is nothing wrong in changing the arguments number in this case.